main.go

```diff
@@ -91,10 +83,7 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	watchNamespace, err := getWatchNamespace()
-	if err != nil {
-		setupLog.Error(err, "unable to get watch namespace, operator running in cluster scoped mode")
-	}
+	watchNamespace, _ := os.LookupEnv(watchNamespaceEnvVar)
```

Comment:
I wonder if we should still print out a log indicating when we are running in cluster scoped mode?

Reply:
@NissesSenap @weisdd @HubertStefanski any preference?

Reply:
I agree with you @pb82 it should be clear which mode the operator is running in.
But i also agree that this shouldn't be an error but rather info.

Reply:
This is already printed in the section below:
https://github.com/grafana-operator/grafana-operator/blob/a622f500863f683d67bb20dc935c54a737a1eda7/main.go#L98-L110

My bad, should've put that in the PR description, I'll update that now.